test/unit_test/passes/onnx/test_perf_tuning.py

```diff
@@ -40,4 +40,19 @@ def test_ort_perf_tuning_pass_with_dynamic_shapes(mock_get_io_config, tmp_path):
     with pytest.raises(TypeError) as e:
         # execute
         p.run(input_model, None, output_folder)
-        assert "ones() received an invalid combination of arguments" in str(e.value)
+    assert "ones() received an invalid combination of arguments" in str(e.value)
```

Comment:
Should the assert in pytest's context?

Reply:
no, the assert should be outside the context

Reply:
otherwise, the assert will be ignored.